nodemon.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
what's the purpose of adding this file?

Reply:
enable auto reloading on twig files updates. this file is not exist but it is defined in `start-backend` command string

![image](https://user-images.githubusercontent.com/15259299/174772738-11411dba-56b9-4444-af14-4f3290a4feec.png)